docs/manager/graphql-reference/schema.graphql

```diff
@@ -1227,6 +1237,12 @@
   vfolder_mounts: [String]
   occupied_slots: JSONString
   requested_slots: JSONString
+
+  """Added in 25.2.0."""
+  image_references: [String]
+
+  """Added in 25.2.0."""
+  vfolder_nodes(filter: String, order: String, offset: Int, before: String, after: String, first: Int, last: Int): VirtualFolderConnection
   num_queries: BigInt
   inference_metrics: JSONString
   kernel_nodes(filter: String, order: String, offset: Int, before: String, after: String, first: Int, last: Int): KernelConnection
```

Comment:
It seems that `kernel_nodes` field is located under the "Added in 25.2.0" annotation, which may cause confusion in the support versioning.

kernel_nodes is a field originally supported since 24.09.0.

One additional question, are all the fields under the "Added in xx.xx.xx" annotation included in that version? Or is only the field directly under the annotation included in that version?

https://github.com/lablup/backend.ai/blob/5b79ec5b8bb51d4ec4d35b2358c8a549cd8bb48d/docs/manager/graphql-reference/schema.graphql#L1232

Reply:
Version annotations currently apply only to the line directly below them, rather than the entire block. Additionally, adding version annotations to all fields of GraphQL types would be impractical and too verbose.